website/scripts/website-build.sh

```diff
@@ -7,7 +12,14 @@ CLONE_DIR=website-preview
 # The product for which we are building the deploy preview
 PRODUCT=nomad
 # Preview mode, controls the UI rendered (either the product site or developer). Can be `io` or `developer`
-PREVIEW_MODE=io
+PREVIEW_MODE=developer
+
+# Get the git branch of the commit that triggered the deploy preview
+# This will power remote image assets in local and deploy previews
+CURRENT_GIT_BRANCH=$VERCEL_GIT_COMMIT_REF
+
+# This is where content files live, relative to the website-preview dir. If omitted, "../content" will be used
+LOCAL_CONTENT_DIR=
```

Comment:
@thiskevinwang I wonder if we want to omit this part if the variable isn't defined 🤔

Reply:
https://github.com/hashicorp/mktg-terraform/pull/41 Should fix this missing var, but this var also only applies to Terraform-projects